tests/execution_engine/split_and_sample/test_sqlalchemy_execution_engine_splitting.py

```diff
@@ -338,7 +349,6 @@ def test_get_split_query_for_data_for_batch_identifiers_for_split_on_date_parts_
 
     assert isinstance(result, sa.sql.Select)
 
-    expected_query_str: str = "SELECT distinct(concat(concat('', CAST(EXTRACT(year FROM column_name) AS VARCHAR)), CAST(EXTRACT(month FROM column_name) AS VARCHAR))) AS concat_distinct_values, CAST(EXTRACT(year FROM column_name) AS INTEGER) AS year, CAST(EXTRACT(month FROM column_name) AS INTEGER) AS month FROM table_name"
```

Comment:
This is testing the query itself, but not that the query is valid for the specified backend. This query raises an error for `sqlite`.